ops/aws/canary/lambda/pkg/scenarios/utils.go

```diff
@@ -149,7 +149,7 @@ func getClient() *client.APIClient {
 
 func getClientV2() clientv2.API {
 	host, port := getClientHostAndPort()
-	return clientv2.New(fmt.Sprintf("http://%s:%d", host, port))
+	return clientv2.New(fmt.Sprintf("http://%s:%d", host, port), clientv2.WithTLS(true), clientv2.WithInsecureTLS(true))
```

Comment:
Is this right? The protocol is still specified as HTTP? The default should still be non-TLS which is what the canary should expect. Is this a left over from when you were doing TLS-by-default?